templates/project/_form.html.twig

```diff
@@ -0,0 +1,11 @@
+<div class="container">
+    <div class="row row-cols-1">
+        <h1>
+            {{ app.request.attributes.get('_route') == 'project_edit' ? 'Edit a ' : 'New ' }}project
```

Comment:
Better use a variable, and during the include use the "include ... with ..." syntaxe : https://twig.symfony.com/doc/3.x/tags/include.html

Reply:
changed by with syntaxe